server/src/test/proto/spine/test/aggregate/events.proto

```diff
@@ -28,10 +28,10 @@ option java_multiple_files = true;
 
 import "spine/annotations.proto";
 import "spine/test/aggregate/project.proto";
-import "spine/test/aggregate/task.proto";
 
 message ProjectCreated {
     ProjectId project_id = 1;
+    string name = 2;
```

Comment:
Looks strange. I would understand the fact that just the `CreateProject` command has a name _and_ ID (which is also suspicious). But why the event has both?